crates/ironclaw_webui_v2_static/static/js/i18n/fr.js

```
@@ -657,36 +657,74 @@ registerPack("fr", {
   "automations.filterLabel": "Filtre d'état des automatisations",
   "automations.filter.all": "All",
   "automations.filter.active": "Actif",
-  "automations.filter.running": "Running",
-  "automations.filter.failures": "Failures",
+  "automations.filter.running": "En cours",
```

Biome (2.5.0)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)